README.md

```diff
@@ -36,7 +36,7 @@ const { DigitalLink, Utils } = require('digital-link.js');
 Add a `<script>` tag to your HTML page:
 
 ```html
-<script src="https://d10ka0m22z5ju5.cloudfront.net/js/digital-link.js/1.2.0/digital-link.js-1.2.0.js"></script>
```

Comment:
I think since this isn't just a bugfix, it should be `1.3.0` 🎉

Reply:
Yes but we don't follow the grammar version anymore if we do this..
I'll see with Dom :)